main.go

```diff
@@ -27,7 +27,7 @@ import (
 
 var (
 	// Version holds the current Gitea version
-	Version = "development"
+	Version = "1.16.9999+dev-main"
```

Comment:
So to dismantle this:

`1.16`
Gitea's version

`.9999`
Uhm.... minor patch is quite high. Guess this is the indication that it's built from development?

`+dev-main`
IIRC it should be `-dev+main`. But I'm not sure about the `+main` part as that is mostly used for build metadata, or something along those lines. Otherwise this could be used as indication that it's development built and then leave the minor patch along.

Reply:
`.9999`: because the `main` branch is always newer than any released version. So I choose the impossible version number 9999. Even during `update check`, the Gitea 1.16.9999 in development won't tell developers that "there is a new version 1.16.8 available".

`+dev-main`: it comes from the current Gitea version format, for exmpale, on try.gitea.io: `1.16.0+dev-782-gde8e3948a`

Reply:
In that sense I think we can use `+dev-local`.